packages/i18nify-js/README.md

```diff
@@ -540,3 +540,50 @@ console.log(getDialCodes()); /*
 console.log(getDialCodeByCountryCode('BR')); // Outputs the dial code for Brazil (+55)
 console.log(getDialCodeByCountryCode('DE')); // Outputs the dial code for Germany (+49)
 ```
+
+### Module 03: Geo Module 🌍
+
+Dive into the digital atlas with the Geo Module 🌍, your go-to source for fetching flag emojis 🏁 and images from around the globe 🌐. Whether you're jazzing up your app with patriotic flair 🎉 or feeding your curiosity about different countries 🤔, this module is your magic carpet ride 🧞‍♂️. Featuring two fantastic functions ✨, it's never been easier to bring a world of flags 🚩 right into your app. Let's unpack these global treasures 🌟:
+
+Powered by the open-source service [FlagCDN](https://flagcdn.com/) 🚀, the Geo Module lets you access high-quality flag images for every country, streamlining the way you showcase flags 🚩. A huge thanks to FlagCDN for its invaluable contribution to the digital world, making it simpler for everyone to connect with cultures and countries globally 🌏.
```

Comment:
Geo module is not powered via flagCDN service. 
We just need to include source of getFlagOfCountry and getFlagForAllCountries in their description. 
Just, image source: flagcdn.com should do IMO